packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -169,7 +171,9 @@
       "props-default-doc:@aws-cdk/aws-lambda.Permission.sourceArn",
       "docs-public-apis:@aws-cdk/aws-lambda.ResourceBindOptions",
       "docs-public-apis:@aws-cdk/aws-lambda.VersionAttributes",
-      "props-physical-name:@aws-cdk/aws-lambda.EventInvokeConfigProps"
+      "props-physical-name:@aws-cdk/aws-lambda.EventInvokeConfigProps",
+      "props-struct-name:@aws-cdk/aws-lambda.IScalableFunctionAttribute",
```

Comment:
Why these exclusions?

Reply:
I looked into it and commented on the initial comment: https://github.com/aws/aws-cdk/pull/8883#discussion_r459899663. Or are you referring to something else?